awscli/examples/secretsmanager/get-secret-value.rst

```diff
@@ -19,3 +19,26 @@ Output::
     }
 
 For more information, see `Retrieve a secret <https://docs.aws.amazon.com/secretsmanager/latest/userguide/retrieving-secrets.html>`__ in the *Secrets Manager User Guide*.
+
+**Example 2: To retrieve the previous secret value**
+
+The following ``get-secret-value`` example gets the current secret value.::

```

Comment:
The only thing that I see that needs correction is that there needs to be a space character here between ``value.`` and ``::`` so that the RST is formatted correctly.

Once that's corrected it'll LGTM.